receiver/stanzareceiver/receiver_test.go

```diff
@@ -72,3 +80,107 @@ func TestHandleConsumeError(t *testing.T) {
 	receiver.Shutdown(context.Background())
 	require.Equal(t, 1, mockConsumer.rejected, "one log entry expected")
 }
+
+func BenchmarkReadLines(b *testing.B) {
```

Comment:
Nit: `BenchmarkReadLine` since it shows the time for a single line not lines.

Reply:
Good call. Updated.